pages/index.js

```diff
@@ -11,15 +13,22 @@ export default function Home({
 }) {
   return (
     <>
-      <Header />
-
-      <StoragesPage
-        storages={storages}
-        onStorage={onStorage}
-        items={items}
-        onItems={onItems}
-      />
-      <WarningPage warningItems={warningItems} />
+      <Header storages={storages} />
+      <StyledDiv>
+        <WarningPage warningItems={warningItems} />
+        <StoragesPage
+          storages={storages}
+          onStorage={onStorage}
+          items={items}
+          onItems={onItems}
+        />
+        <AddStorageForm onStorage={onStorage} storages={storages} />
+      </StyledDiv>
     </>
   );
 }
+
+const StyledDiv = styled.div`
```

Comment:
Bei Gelegenheit würde ich auch hier nochmal die Eigenschaften vom Header & von der StorageCard anpassen, dann musst du die Elemente garnicht in ein Div packen :)

Reply:
Hey, leider klappt es ohne dieses div gar nicht :/ ich hab da einiges probiert und nur mit dem div bleiben die Abstände richtig